llvm/include/llvm/Frontend/OpenMP/OMPIRBuilder.h

```diff
@@ -99,14 +100,20 @@ class OpenMPIRBuilderConfig {
   /// expanded.
   std::optional<bool> IsGPU;
 
-  // Flag for specifying if offloading is mandatory.
+  /// Flag for specifying if LLVMUsed information should be emitted.
+  std::optional<bool> EmitLLVMUsed;
```

Comment:
This is a generic sounding flag that guards a very very specific thing.
Is there ever a use case where the flag is not set (and it is checked)?

Reply:
I've changed the name to `EmitLLVMUsedMetaInfo`, please let me know if you have a better name.

I don't think we have such a use case. I've updated the current code to check the flag is set before checking it.